src/Commands/MoveCommand.php

```diff
@@ -30,7 +30,7 @@ public function handle()
         $class = $this->renameClass();
         if (! $inline) $view = $this->renameView();
 
-        $test = $this->option('test');
+        $test = $this->option('test') || $this->option('pest');
```

Comment:
I see you're adding support for pest here, but does this actually support renaming the pest test? Have you tested this yourself locally? It would appear that this triggers the normal test renaming code later on in this file. Can you confirm? Otherwise, support the rename?

Reply:
This same comment applies to the other non-make commands.

Reply:
I'm guessing it's because the pest test that is generated is the same in terms of the file name, however, I just want to make sure you've tried all of these locally and can confirm first. Thanks.